THEMING.md

```diff
@@ -0,0 +1,142 @@
+
+The Container scheme represents a single configurable entity that is applicable to all themeable components. A container scheme consists of all of the subsystem schemes.
+
+Theming extensions are component extensions that, when invoked with a default container scheme, will configure a component to match the design system's defaults. When provided with subsystem schemes via a container scheme, the extension will map the subsystem scheme's values to the component’s parameters.
+
+### Sensible defaults, yet highly configurable
+
+By default, we try and give our components sensible values for all of their customizable properties, things such as `backgroundColor` or `titleFont`. You can use these defaults as a baseline, but we encourage you to theming your components to match your brand style.
+
+### Schemes
+
+<ul class="icon-list">
+<li class="icon-list-item icon-list-item--link"><a href="Container/">Container scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Color/">Color scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Shape/">Shape scheme</a></li>
+<li class="icon-list-item icon-list-item--link"><a href="Typography/">Typography scheme</a></li>
+</ul>
+
+#### Container scheme
+
+The Container Scheme represents a single configurable entity that is applicable to all themeable Components. A Container Scheme consists of all of the Subsystem Schemes.
+
+## Examples
+
+### Using a subsystem scheme
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+import MaterialComponents.MaterialColorScheme
+
+let colorScheme = MDCSemanticColorScheme(defaults: .material201804)
+// Configure custom properties to match your branch
+colorScheme.backgroundColor = .black
+colorScheme.primaryColor = .yellow
+colorScheme.secondaryColor = .red
+```
+
+#### Objective-C
+
+```objc
+#import "MaterialColorScheme.h"
+
+MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+// Configure custom properties to match your branch
+colorScheme.backgroundColor = UIColor.blackColor;
+colorScheme.primaryColor = UIColor.yellowColor;
+colorScheme.secondaryColor = UIColor.redColor;
+```
+<!--</div>-->
+
+### Creating a container scheme
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+import MaterialComponents.MaterialColorScheme
+import MaterialComponents.MaterialTypographyScheme
+import MaterialComponents.MaterialShapeScheme
+
+let containerScheme = MDCContainerScheme()
+containerScheme.colorScheme = myColorScheme
+containerScheme.typographyScheme = myTypographyScheme
+containerScheme.shapeScheme = myShapeScheme
+```
+
+#### Objective-C
+
+```objc
+#import "MaterialColorScheme.h"
```

Comment:
Missing import for `MaterialContainerScheme.h`

Reply:
Done